boot/commands.go

```diff
@@ -0,0 +1,104 @@
+package boot
+
+import (
+	"github.com/wakeful-deployment/operator/directory"
+	"github.com/wakeful-deployment/operator/global"
+	"github.com/wakeful-deployment/operator/node"
+	"time"
+)
+
+func DetectOrBootConsul(state *State) error {
+	// TODO
+	return nil
+}
+
+func LoadStateFromFile(path string) *State {
+	state, err := ReadStateFromConfigFile(path)
+
+	if err != nil {
+		global.Transition(global.NewState(global.ConfigFailed, err))
+		return nil
+	}
+
+	return state
+}
+
+func Boot(state *State) {
+	var err error
+
+	global.Transition(global.NewState(global.Booting, nil))
```

Comment:
I want to make a `TransitionState(s State, e error)` that will call `NewState` internally.